### PR TITLE
Updated npm-faq(7) -- Frequently Asked Questions

### PR DESCRIPTION
**Pull request by AndSheCodes2**
Adding empathy to FAQ.
Fixes: #10022

### CODE REVIEW

**Comment by aredridel** [comment]
<3 this.

**Comment by steveklabnik** [comment]
👏👏👏👏👏👏👏👏👏

**Comment by ashleygwilliams** [comment]
hey! travis is breaking on 0.8 ... it looks like it is *always* breaking... thoughts, @aredridel ?

**Comment by TaMe3971** [comment]
I just created a PR #10023

**Comment by nchase** [comment]
👍👌

File: doc/misc/npm-faq.md [hunk 1]
```diff
@@ -7,18 +7,19 @@ npm-faq(7) -- Frequently Asked Questions
...
 
 ## Why didn't it work?
 
 I don't know yet.
 
-Read the error output, and if you can't figure out what it means,
-do what it says and post a bug with all the information it asks for.
+Try reading the error output first, ensure this is a true npm issue and not a package issue. If you are having an issue with a package dependency, please submit your error to that particular package maintainer. 
+
+For any npm issues, try following the instructions, or even retracing your steps.  If the issue continues to persist, submit a bug with the steps to reproduce, please include the operating system you are working on, along with the error you recieve.
```
Comment (jzaefferer) [inline doc/misc/npm-faq.md]: Double spaces after the first sentence.

File: doc/misc/npm-faq.md [hunk 2]
```diff
@@ -7,18 +7,19 @@ npm-faq(7) -- Frequently Asked Questions
 
     npm config set viewer browser
 
-to open these documents in your default web browser rather than `man`.
+Open the following documents in your default web browser rather than `man`.
```
Comment (jzaefferer) [inline doc/misc/npm-faq.md]: This doesn't make sense to me. It seems to refer to documents outside of this one, not the "following", i.e. anything below.

Comment (jzaefferer) [inline doc/misc/npm-faq.md]: "This will open these documents ..."

**Comment by jzaefferer** [comment]
Looks good to me now.

**Comment by iarna** [comment]
@ashleygwilliams Yeah, 0.8 always fails =/ Some of our dev deps dropped support for 0.8 and we haven't gone through and fixed that yet. (Plus undoubtably some of our own tests have lost 0.8 support in the time we haven't had CI coverage for it.)

**Comment by iarna** [comment]
Thank you for this! It landed in 3.3.10!

**Pull request closed without merging**